app/UI/app.tsx

```diff
@@ -397,12 +407,16 @@ export class AppComponent extends Component<AppProps> {
                                     event: e as Parsed_Event<NostrKind.TEXT_NOTE | NostrKind.Long_Form>,
                                     lamport: getTags(e).lamport_timestamp,
                                     type: "text",
+                                    is_deleted: app.database.isDeleted(
```

Comment:
As commented below, let's filter out deleted messages instead of passing them down